textdb/textdb-perftest/src/main/java/edu/uci/ics/textdb/perftest/sample/SampleExtraction.java

```diff
@@ -40,12 +39,28 @@
     
     public static final String PROMED_SAMPLE_TABLE = "promed";
         
-    public static final String promedFilesDirectory = "../textdb-perftest/sample-data-files/promed/";
-    public static final String promedIndexDirectory = "../textdb-perftest/index/standard/promed/";
-    
-    
+    public static String promedFilesDirectory;
+    public static String promedIndexDirectory;
+    public static String sampleDataFilesDirectory;
+
+    static {
+        try {
+            // Finding the absolute path to the sample data files directory and index directory
+            promedFilesDirectory = Paths.get(SampleExtraction.class.getResource("/sample-data-files/promed")
+                    .toURI())
+                    .toString();
+            promedIndexDirectory = Paths.get(SampleExtraction.class.getResource("/index/standard")
+                    .toURI())
+                    .toString() + "/promed";
+            sampleDataFilesDirectory = Paths.get(SampleExtraction.class.getResource("/sample-data-files")
+                    .toURI())
+                    .toString();
+        }
+        catch(URISyntaxException e) {
+            e.printStackTrace();
```

Comment:
I will let @zuozhiw be the reviewer.  If he approves, you @kishore-narendran can merge it.